### PR TITLE
added multi-docker-pull and edited license

### PR DESCRIPTION
**Pull request by noahgildersleeve**
fixed pep8 syntax issues

### CODE REVIEW

File: scripts/multi-docker-pull/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# multi-docker-pull
+Pull multiple Docker images at once and then tarball them
+
+This script will create a group of worker threads that will simultaneously pull docker images to your local docker instance.
```
Comment (irishgordo) [inline scripts/multi-docker-pull/README.md:4]:
```suggestion
This script will create a group of worker threads that will simultaneously pull docker images to your local docker instance.

# requirements
- script tested with `docker==7.0.0` [pip package](https://pypi.org/project/docker/)
- script tested with `pigz 2.6`, [pigz](https://zlib.net/pigz/)
- script tested with Python 3.10
```

**Review by irishgordo — CHANGES_REQUESTED** [review]
this looks great and works well :smile: :+1: 

since we're not bundling in a requirements.txt with this, perhaps adding a little note to the readme about the docker pypi package, pigz, and python could help be beneficial :smile:

File: scripts/multi-docker-pull/README.md (hunk 1 above)
Comment (noahgildersleeve) [reply scripts/multi-docker-pull/README.md]: That's a good point. I've been meaning to add a requirements.txt to this, but I keep forgetting. I'll do it later and commit this for now. Thanks for taking a look.

**Review by irishgordo — APPROVED** [review]
awesome - looks good to me :smile: :+1: - lgtm

**Pull request merged**